samples/net/wpan_serial/src/main.c

```diff
@@ -523,6 +524,12 @@ void main(void)
 		return;
 	}
 
+	ret = usb_enable(NULL);
```

Comment:
Should USB be enabled from CDC ACM?

Reply:
USB is now always enabled from the samples, not the classes, since @emob-nordic introduced the change.